src/resolvers-cassandra/Settings/queries.js

```diff
@@ -2,12 +2,58 @@
 
 const Promise = require('promise');
 const facebookAnalyticsClient = require('../../clients/facebook/FacebookAnalyticsClient');
+const cassandraConnector = require('../../clients/cassandra/CassandraConnector');
+
+function cassandraRowToSite(row) {
+  return {
+    name: row.sitename,
+    properties: {
+      targetBbox: row.geofence,
+      defaultZoomLevel: row.defaultzoom,
+      logo: row.logo,
+      title: row.title,
+      defaultLocation: row.geofence,
+      // TODO: Ask what the following commented properties map to:
+    // storageConnectionString: String,
+    // featuresConnectionString: String,
+    // mapzenApiKey: String,
+    // fbToken: String,
+      supportedLanguages: row.languages
+    }
+  };
+}
 
 /**
  * @param {{siteId: string}} args
  * @returns {Promise.<{runTime: string, sites: Array<{name: string, properties: {targetBbox: number[], defaultZoomLevel: number, logo: string, title: string, defaultLocation: number[], storageConnectionString: string, featuresConnectionString: string, mapzenApiKey: string, fbToken: string, supportedLanguages: string[]}}>}>}
  */
 function sites(args, res) { // eslint-disable-line no-unused-vars
+  const startTime = Date.now();
+
+  return new Promise((resolve, reject) => {
+    const siteId = args.siteId;
+    if (!siteId) {
+      return reject('No site id to fetch specified');
+    }
+
+    const siteById = 'SELECT * FROM fortis.sitesettings WHERE id = ?';
+    cassandraConnector.executeQuery(siteById, [siteId])
+    // .catch(err => reject(err))
+    .then(rows => {
+      if (rows.length < 1) {
+        return reject(`Could not find site with id ${siteId}`);
+      }
+      if (rows.length > 1) {
+        return reject(`Got more than one (${rows.length}) site with id ${siteId}`);
```

Comment:
Good, point! But I think we should be safe since it will look more like this:

```
Got more than one (2) site with id d4d294ec-5b2c-4674-8f3f-e1eb7b8dbb47
```